hail/src/test/scala/is/hail/expr/ir/TableIRSuite.scala

```diff
@@ -309,64 +309,55 @@ class TableIRSuite extends HailSuite {
   @DataProvider(name = "join")
   def joinData(): Array[Array[Any]] =
     for {
-      l <- leftPartitioners
-      r <- rightPartitioners
+      lParts <- Array[Integer](1, 2, 3)
+      rParts <- Array[Integer](1, 2, 3)
       (j, p) <- joinTypes
       leftProject <- Seq[Set[Int]](Set(), Set(1), Set(2), Set(1, 2))
       rightProject <- Seq[Set[Int]](Set(), Set(1), Set(2), Set(1, 2))
       if !leftProject.contains(1) || rightProject.contains(1)
-    } yield Array[Any](l, r, j, p, leftProject, rightProject)
+    } yield Array[Any](lParts, rParts, j, p, leftProject, rightProject)
 
   @Test(dataProvider = "join")
   def testTableJoin(
-    leftPart: RVDPartitioner,
-    rightPart: RVDPartitioner,
+    lParts: Int,
+    rParts: Int,
     joinType: String,
     pred: Row => Boolean,
     leftProject: Set[Int],
     rightProject: Set[Int]
   ) {
-    implicit val execStrats = ExecStrategy.interpretOnly
+    implicit val execStrats = ExecStrategy.allRelational
```

Comment:
This should be the default I think. Like if you delete this line you should pick up `allRelational` from an outer implicit.

Reply:
Ah, true.